packages/cashier/jest.config.js

```diff
@@ -6,19 +6,19 @@ module.exports = {
     moduleNameMapper: {
         '\\.s(c|a)ss$': '<rootDir>/../../__mocks__/styleMock.js',
         '^.+\\.svg$': '<rootDir>/../../__mocks__/styleMock.js',
-        '^Stores/(.*)$': '<rootDir>/src/Stores/$1',
-        '^Constants/(.*)$': '<rootDir>/src/Constants/$1',
+        '^Stores/(.*)$': '<rootDir>/src/stores/$1',
+        '^Constants/(.*)$': '<rootDir>/src/constants/$1',
         '^_common/(.*)$': '<rootDir>/src/_common/$1',
-        '^Config/(.*)$': '<rootDir>/src/Config/$1',
-        '^Components/(.*)$': '<rootDir>/src/Components/$1',
-        '^Utils/(.*)$': '<rootDir>/src/Utils/$1',
-        '^Containers/(.*)$': '<rootDir>/src/Containers/$1',
+        '^Config/(.*)$': '<rootDir>/src/config/$1',
+        '^Components/(.*)$': '<rootDir>/src/components/$1',
+        '^Utils/(.*)$': '<rootDir>/src/utils/$1',
+        '^Containers/(.*)$': '<rootDir>/src/containers/$1',
```

Comment:
can we add '^Pages/(.*)$': '<rootDir>/src/pages/$1', as part of these changes, as we will be working on this branch ?

Reply:
Yes, will add that. Thank you 👍